de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/process/M_ShipmentSchedule_EnqueueSelection.java

```diff
@@ -54,8 +54,8 @@
 		implements IProcessPrecondition
 {
 
-	@Param(parameterName = "IsUseQtyPicked", mandatory = true)
-	private boolean isUseQtyPicked;
+	@Param(parameterName = "Quantity", mandatory = true)
```

Comment:
with Quantity as param name, i'd expect a quantity..please call it quantitytype etc